core/os/android/adb/device.go

```diff
@@ -114,18 +113,18 @@ func Devices(ctx context.Context) (DeviceList, error) {
 	return out, nil
 }
 
-func SetupPrereleaseDriver(ctx context.Context, d Device, p *android.InstalledPackage) (app.Cleanup, error) {
+func SetupPrereleaseDriver(ctx context.Context, d Device, packageName string) (app.Cleanup, error) {
```

Comment:
I'm not sure about this part of the change. Passing an entire InstalledPackage here makes this harder to misuse.